CMakeLists.txt

```diff
@@ -1,19 +1,18 @@
 cmake_minimum_required(VERSION 3.10)
 
+project(chip8)
+
 file(GLOB SRC src/*.cc)
 file(GLOB INC inc/*.h)
 
-include(deps/CMakeLists.txt)
-project(chip8)
-
-find_package(sdl2)
+include(${CMAKE_BINARY_DIR}/conanbuildinfo.cmake)
+conan_basic_setup()
 
 add_executable(${PROJECT_NAME} ${SRC})
 target_include_directories(
 	${PROJECT_NAME}
 		PRIVATE ${INC}
-		PRIVATE ${SDL2_INCLUDE_DIRS}
 )
 
-
-target_link_libraries(${PROJECT_NAME} PRIVATE ${SDL2_LIBRARIES})
\ No newline at end of file
+set_property(TARGET ${PROJECT_NAME} PROPERTY CXX_STANDARD 17)
```

Comment:
Can you add more than 1 version of C++, Iike C++ 11 too? Just curious

Reply:
Not that I know of, but you can use a combination of conditionals with an argument passed to cmake to change versions.